CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [3.13.0] - 2023-03-28
 
 ### Added
+- Add Client UTC Offset inside join message which is sent over signalling channel.
```

Comment:
Can make this builder friendly: "Send client UTC offset with attendee `JOIN` frame over signalling channel."

Reply:
Changed. Thanks.